man/kernel-install.xml

```diff
@@ -209,8 +213,10 @@
         </term>
           <listitem>
             <para>The content of this file specifies the machine identification
-            <replaceable>MACHINE-ID</replaceable>. If it cannot read <filename>/etc/machine-id</filename>,
-            kernel-install will use "Linux" as the machine ID instead.</para>
+            <replaceable>MACHINE-ID</replaceable>. If <filename>/etc/machine-id</filename>
+            cannot be read or is temporary (backed by a file on <constant>tmpfs</constant>),
+            or <filename>$BOOT/Default</filename> exists, <command>kernel-install</command>
+            will use <literal>Default</literal> instead.</para>
```

Comment:
maybe say "… the literal string <literal>Default</literal>", i.e. underline we really mean this as literal string, not as a placeholder for something

Reply:
went with "the literal Default", but `<literal>` already wraps the contents in `"`, so it's laying it on a bit thick, I think